dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -368,11 +414,19 @@ def _get_dataset(value):
                 input_names_list.append(input_name)
                 input_datasets_list.append(_get_dataset(input_value))
 
-                # TODO(klecki): Do we want all Python-only ES parameters to be overridable here?
-                if is_dataset_only or input_value.layout is None:
-                    input_layouts_list.append(self._get_layout_from_pipeline(input_name, name_es_map))
+                if is_dataset_only:
+                    # Set the defaults used in lookup
+                    as_input = experimental.Input(input_value, layout=None, batch=False)
```

Comment:
what's the value of `layout=None`? Isn't it the same as not providing anything?

Reply:
Yes, but I want to be explicit here.